frontends/web/src/common/Annotation/CreateInterface.js

```diff
@@ -525,7 +535,12 @@ class CreateInterface extends React.Component {
           this.setState(
             { task: result, taskCode: result.task_code },
             function () {
-              this.state.task.selected_round = this.state.task.cur_round;
+              this.setState((prevState) => ({
```

Comment:
@ktirumalafb don't we actually need to revert these setsState changes? setState is asynchronous, so I don't think that there is a guarantee that this.state.task.selected_round will be updated fast enough. Do you think that I have the correct understanding, or am I missing something? Thanks.

Reply:
I think you are definitely right about the state change becoming asynchronous, however I think usually you're never supposed to mutate state directly. Since the part after these set states is just one function call `this.context.api.convertToModelIO`, we could probably just use callbacks (add a `.then()`) after the `setState` calls